main.py

```diff
@@ -649,40 +645,51 @@ def update_outs_parallel(
         "--finetune_max_epochs",
         type=int,
         default=1000,
-        help="Run this many passes over training data when doing global optimization; no means skip GO",
+        help="Run this many passes over training data when doing finetuning; no means skip",
```

Comment:
"0 means skip"?

Reply:
This means, that we do not finetune. I can write "no means skip finetune"

Reply:
It's an `int` argument. It can't be 'no'

Reply:
Agree, I'll fix it